art.taunoerik.tauno-serial-plotter.yml

```
@@ -0,0 +1,37 @@
+app-id: art.taunoerik.tauno-serial-plotter
+runtime: org.kde.Platform
+runtime-version: '5.15-21.08'
+sdk: org.kde.Sdk
+command: tauno-serial-plotter.py
+
+finish-args:
+  - --share=ipc
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=network
+  - --device=all
+
+modules:
+  - python3-requirements.json
+
+  - name: tauno-serial-plotter
+    buildsystem: simple
+    build-commands:
+      - install -D src/tauno-serial-plotter.py /app/bin/tauno-serial-plotter.py
+      - install -Dm644 art.taunoerik.tauno-serial-plotter.desktop /app/share/applications/Tauno-serial-plotter.desktop
```

1. if it as the proper name, just insall 
2. I see you made a new release that would have been the time to add the destkop file into that release. A missing desktop file a GUI application for linux IS a bug.
```suggestion
      - install -Dm644 -t /app/share/applications/ art.taunoerik.tauno-serial-plotter.desktop
```